utils/installers/install.sh

```diff
@@ -0,0 +1,495 @@
+    fi
+}
+
+# Expose the hf CLI by linking or copying the console script from the virtualenv
+expose_cli_command() {
+    log_info "Linking hf CLI into $BIN_DIR..."
+
+    local source_cli="$VENV_DIR/bin/hf"
+    if [ ! -x "$source_cli" ]; then
+        log_error "hf command not found in the virtual environment at $source_cli"
+        log_error "Verify that huggingface_hub[cli] installed correctly."
+        exit 1
+    fi
+
+    local link_method=""
+    if ln -sf "$source_cli" "$BIN_DIR/hf" 2>/dev/null; then
+        link_method="symlink"
+    else
+        if cp "$source_cli" "$BIN_DIR/hf" 2>/dev/null; then
+            link_method="copy"
+        else
+            log_error "Failed to place hf command in $BIN_DIR (tried symlink and copy)."
+            exit 1
+        fi
+    fi
+
+    chmod +x "$BIN_DIR/hf"
+
+    if [ "$link_method" = "symlink" ]; then
+        log_info "hf available at $BIN_DIR/hf (symlink to venv)"
+    else
+        log_info "hf available at $BIN_DIR/hf"
+    fi
+    log_info "Run without touching PATH: env PATH=\"$BIN_DIR:\$PATH\" hf --help"
+}
+
+# Update PATH if needed
+update_path() {
+    local shell_rc=""
+    local -a shell_rc_candidates=()
+
+    # Broaden shell detection and guidance for PATH propagation
+    case "$SHELL" in
+        */bash)
+            shell_rc_candidates=()
+            shell_rc_candidates+=("$HOME/.bashrc")
+            shell_rc_candidates+=("$HOME/.bash_profile")
+            shell_rc_candidates+=("$HOME/.profile")
+            ;;
+        */zsh)
+            shell_rc_candidates=()
+            shell_rc_candidates+=("$HOME/.zshrc")
+            shell_rc_candidates+=("$HOME/.zprofile")
+            ;;
+        */fish)
+            shell_rc_candidates=()
+            if command -v fish >/dev/null 2>&1; then
+                if fish -c "contains \"$BIN_DIR\" \$fish_user_paths" >/dev/null 2>&1; then
+                    log_info "$BIN_DIR already present in fish_user_paths"
+                    UPDATED_FISH_PATH="true"
+                    return
+                elif fish -c "set -Ux fish_user_paths \"$BIN_DIR\" \$fish_user_paths" >/dev/null 2>&1; then
+                    UPDATED_FISH_PATH="true"
+                    log_success "Added $BIN_DIR to fish_user_paths"
+                    log_info "Apply it now with: set -Ux fish_user_paths $BIN_DIR \$fish_user_paths"
+                    return
+                else
+                    log_warning "Could not update fish_user_paths automatically."
+                fi
+            fi
+            ;;
+        *)
+            shell_rc_candidates=()
+            shell_rc_candidates+=("$HOME/.profile")
+            ;;
+    esac
+
+    if [[ ":$PATH:" != *":$BIN_DIR:"* ]]; then
+        if [ "$SKIP_PATH_UPDATE" = "true" ]; then
+            log_info "Skipping PATH update (--no-modify-path)."
+            return
+        fi
+
+        log_warning "$BIN_DIR is not in your PATH"
```

Comment:
```suggestion
        log_info "$BIN_DIR is not in your PATH"
```

maybe an info? (it's normal once first script launch)